### ISSUE TITLE
Feature: user-configurable contrast ratio

### ISSUE THREAD

**Issue by sgregson** [sgregson/accessible-color-matrix#2]
WCAG contrast ratio values are `4.5:1` and `3:1`

My fork exposes this ability in a hacky way, with https://sgregson.github.io/accessible-color-matrix/4.5/ and https://sgregson.github.io/accessible-color-matrix/3/ .

## Goals

- [ ] make paletteRatio a configurable parameter. `3` and `4.5` should have labels (per WCAG / 508) but user input should be accepted
- [ ] make paletteRatio serializable via a `share` link, see #3
